domains/transfer/Transfer.API/API.IntegrationTests/Factories/TransferAgreementsApiWebApplicationFactory.cs

```diff
@@ -162,6 +149,24 @@ public HttpClient CreateUnauthenticatedClient()
         client.DefaultRequestHeaders.Authorization =
             new AuthenticationHeaderValue("Bearer", GenerateToken(sub: sub, tin: tin, name: name, actor: actor));
         client.DefaultRequestHeaders.Add("EO_API_VERSION", apiVersion);
+
+        return client;
+    }
+
+    public HttpClient CreateAuthenticatedClient(IProjectOriginWalletService poWalletServiceMock, string sub, string tin = "11223344", string name = "Peter Producent",
```

Comment:
why is this a diff?

Reply:
Because I added IProjectOriginWalletService poWalletServiceMock